cmd/users.go

```diff
@@ -22,9 +22,12 @@ func (d DeisCmd) UsersList(results int) error {
 		return err
 	}
 
-	d.Printf("=== Users%s", limitCount(len(users), count))
+	d.Printf("=== Users (* admin)%s", limitCount(len(users), count))
```

Comment:
`* admin` doesn't really explain that names prefixed with * means they're an admin user. Would you mind elaborating on this instead in the docstring in parser/users.go?